### PR TITLE
constants, testmap: Set TEST_DEFAULT_OS to fedora-35

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
Hmm, I guess `fedora-35/mobile` needs to be in testmap on main in order to be triggerable here, right?  But `fedora-35` should run the pixeltests now, let's see.

File: lib/testmap.py [hunk 1]
```diff
@@ -27,16 +27,16 @@
...
             'debian-stable',
             'debian-testing',
             'ubuntu-2004',
             'ubuntu-stable',
             'fedora-34',
-            'fedora-34/devel',
-            'fedora-34/firefox',
-            'fedora-34/mobile',
-            'fedora-35',
+            TEST_OS_DEFAULT,
```
Comment (allisonkarlitskaya) [inline lib/testmap.py:36]: I'd maybe avoid this particular one and just leave it open.

File: lib/testmap.py [hunk 2]
```diff
@@ -27,16 +27,16 @@
...
+            TEST_OS_DEFAULT + '/devel',
+            TEST_OS_DEFAULT + '/firefox',
+            TEST_OS_DEFAULT + '/mobile',
```
Comment (allisonkarlitskaya) [inline lib/testmap.py:39]: ...and for these three, please use f-strings.

File: lib/testmap.py [hunk 3]
```diff
@@ -52,28 +52,25 @@
             'rhel-8-5',
             'rhel-8-5-distropkg',
             'centos-8-stream',
-            'fedora-34/container-bastion',
+            TEST_OS_DEFAULT + '/container-bastion',
```
Comment (allisonkarlitskaya) [inline lib/testmap.py:55]: ditto.

(and all the others under)

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

File: lib/testmap.py (hunk 1 above)
Comment (mvollmer) [reply lib/testmap.py]: Ok, done.

File: lib/testmap.py (hunk 2 above)
Comment (mvollmer) [reply lib/testmap.py]: Done.

File: lib/testmap.py (hunk 3 above)
Comment (mvollmer) [reply lib/testmap.py]: Done.

**Comment by mvollmer** [comment]
Summary of pixel test failures with Fedora 35:
- virbr0 isn't listed anymore on the networking page
- setroubleshoot text has changed, affecting sort order of solutions
- zram is offered for creating vgroups on and stuff
- /dev/vda4 is offered as well, empty partition on root disk?
- system info says "fedora-35" instead 34 :-)

I check the storage things, ideally zram0 and vda4 wouldn't be offered.

**Comment by mvollmer** [comment]
> Summary of pixel test failures with Fedora 35:

Also, fedora-35 has a failed service in the health card.  I check that as well.

**Comment by mvollmer** [comment]
> Also, fedora-35 has a failed service in the health card. I check that as well.

This is rhsm.service, due to https://bugzilla.redhat.com/show_bug.cgi?id=1995465.  It should be ignored in the pixel test, along with the rest of the health card... hmm.

**Comment by mvollmer** [comment]
> I check the storage things, ideally zram0 and vda4 wouldn't be offered.

Let's just accept them. vda4 is flagged as "prep", which is some powerpc boot thingy, zram should be ignored in general, but right now it isn't so let's live with it.

**Comment by mvollmer** [comment]
Nice, machines and podman don't need any changes to pixeltests.

**Comment by mvollmer** [comment]
Updated pixels in https://github.com/cockpit-project/cockpit/pull/16716

**Comment by mvollmer** [comment]
fedora-35/devel@cockpit-project/cockpit failure needs investigation: https://logs.cockpit-project.org/logs/pull-2705-20211210-130557-c6eac50e-fedora-35-devel-cockpit-project-cockpit/log.html#109-2

**Comment by mvollmer** [comment]
> fedora-35/devel@cockpit-project/cockpit failure needs investigation:

TestUpdatesSubscriptions.testNoSubOsRepo fails on */devel because packagekit is not pre-loaded there and thus the health card stays empty. I'll fix that by explicitly loading the `/updates` page in the test.

**Comment by mvollmer** [comment]
> > fedora-35/devel@cockpit-project/cockpit failure needs investigation:
> 
> TestUpdatesSubscriptions.testNoSubOsRepo fails on */devel because packagekit is not pre-loaded there and thus the health card stays empty. I'll fix that by explicitly loading the `/updates` page in the test.

We agreed to just skip all of TestUpdatesSubscriptions on fedora-35, https://github.com/cockpit-project/cockpit/pull/16720

**Review by allisonkarlitskaya — APPROVED** [review]
Looks fine to me, if it lands in lockstep with the pixel test changes on cockpit repo.

**Pull request merged**